core/error/error_macros.h

```diff
@@ -730,6 +730,16 @@ void _err_flush_stdout();
 	} else                                                                                        \
 		((void)0)
 
+/**
+ * Warns about `m_msg` only when verbose mode is enabled.
+ */
+#define WARN_VERBOSE(m_msg)               \
```

Comment:
I'm ok with this, but typically need an opinion from the core team about these additions to macros.